README.md

```diff
@@ -12,25 +23,13 @@
 |Nomad|0.12.3 or newer|0.12.3 or newer|
 
 ## Usage
-
-### Starting datastack
-Run
-```makefile
-make up
-```
-in your terminal to provision the datastack. When the process is finished you can then open up access to Redash, Presto, and MinIO by running
-```makefile
-make connect-to-all
-```
-which will open 3 terminal windows each with a connection to one of the components. Alternatively you can run `make proxy-redash`, `make proxy-presto`, and `make proxy-minio` to open individual proxy connections.
-
 ### Requirements
 #### Required modules
 - [MinIO](https://github.com/fredrikhgrelland/terraform-nomad-minio)
 - [Hive](https://github.com/fredrikhgrelland/terraform-nomad-hive)
 
 #### Required software
-- [GNU make](https://man7.org/linux/man-pages/man1/make.1.html)
+See [prerequisites](template_README.md#install-prerequisites)
```

Comment:
We need to add info regarding module specific software requirement before providing link to the more generic ones.
For example we need ```consul``` binary here

Reply:
In my head there are no module-specific requirements, and all requirements are inherited from the template. Or am I thinking about this the wrong way?

Reply:
@Neha-Sinha2305 any input? My thought was to not change it, do agree, or no?

Reply:
I don't think that we need an additional section on consul. Also, saw this comment from Nikita so may be we need to add the consul binary requirement in the vagrant-hashistack-template project anyway:
https://github.com/fredrikhgrelland/terraform-nomad-redash/issues/12#issuecomment-683728415